src/dash-table/dash/DataTable.js

```diff
@@ -483,7 +484,11 @@ export const propTypes = {
         link_target: PropTypes.oneOfType([
             PropTypes.string,
             PropTypes.oneOf(['_blank', '_parent', '_self', '_top'])
-        ]).isRequired
+        ]),
+        /**
+         * (default: False)  If True, html may be used in markdown cells
```

Comment:
```suggestion
         * (default: False)  If True, html may be used in markdown cells
         * Be careful enabling html if the content being rendered can come
         * from an untrusted user, as this may create an XSS vulnerability.
```

I think it's worth mentioning... The general pattern here is:
- A malicious user enters information that gets saved in a database or file
- An authorized user logs into the app
- If the malicious content is rendered as html, it can contain JavaScript that executes
- This JavaScript is executed with the permissions of the authorized user, so can make requests to the server to read and/or change information that public users normally are not allowed to access.

It obviously depends on the details of the app but there can be a lot of ways - often unexpected - that a malicious user can get data into a system. So typically the easiest way to plug this attack is by not rendering content directly as html.

Anyway this PR looks great - excellent tests!

Reply:
Thanks for the info on XSS vulnerabilities.  Will this extended description be added to the docs as well?

Reply:
These docstrings automatically make it into the reference part of the docs. If we want to write up a longer description of these issues somewhere we can do that as well. FWIW this is the reason for the "dangerously" in https://github.com/plotly/dash-dangerously-set-inner-html as well - and the readme there talks a little more about this with a link that gives more info.